repo/game.py

```diff
@@ -0,0 +1,73 @@
+import datetime
+from typing import List
+
+from sqlalchemy import Column, Integer, String, DateTime
+from sqlalchemy.orm import Session
+
+from board import Stone, BoardConfig
+
+
+class Game:
+    __tablename__ = "game"
+
+    id: Column(Integer, primary_key=True)
+    winner = Column(String)
+    total_size = Column(Integer)
+    board_config = Column(String)
+    black_player = Column(String)
+    white_player = Column(String)
+    created_time = Column(DateTime)
+
+    def __init__(self,
+                 winner: str,
+                 board_config: BoardConfig,
+                 black_player: str,
+                 white_player: str,
+                 logs: List[Stone]):
+        self.winner = winner,
+        self.total_size = len(logs),
+        self.board_config = str(board_config),
```

Comment:
Since the result of this string start with "BoardConfig(...)", convert this dataclass to JSON string seems better